common/src/cams/test-utilities/mock-data.ts

```diff
@@ -324,6 +326,34 @@ function getDateBeforeToday() {
   return faker.date.past();
 }
 
+function getCamsSession(override: Partial<CamsSession> = {}): CamsSession {
+  return {
+    user: { name: 'Mock Name' },
+    apiToken: getJwt(),
+    provider: 'mock',
+    validatedClaims: { claimOne: '' },
+    ...override,
+  };
+}
+
+function getJwt(): string {
+  const header = '{"typ":"JWT","alg":"HS256"}';
+  const now = new Date();
+  const futureExpirationDate = new Date(
+    now.getFullYear(),
+    now.getMonth(),
+    now.getDate(),
+    now.getHours() + 1,
+    now.getMinutes(),
+  );
+  const payload = `{"iss":"http://fake.issuer.com","sub":"user@fake.com","aud":"fakeApi","exp":${futureExpirationDate.valueOf()}}`;
+  const signature = 'fake-signature-text';
+  const encodedHeader = Buffer.from(header, 'binary').toString('base64');
```

Comment:
We could use the JWT library from Auth0 to sign a mock JWT here.

Reply:
No we cannot. The jsonwebtoken package is dependent on safe-buffer which CANNOT be used in FE code.